airflow/hooks/http_hook.py

```diff
@@ -75,20 +75,24 @@ def get_conn(self, headers=None):
                 try:
                     session.headers.update(conn.extra_dejson)
                 except TypeError:
-                    self.log.warning('Connection to %s has invalid extra field.', conn.host)
+                    self.log.warning(
+                        'Connection to %s has invalid extra field.', conn.host)
```

Comment:
I think this line is unrelated and unnecessary change? So as somewhere else in this PR

Reply:
Thanks, I'll remove them